translations/en.i18n.json

```diff
@@ -6,7 +6,9 @@
       "no_longer_live": "Broadcast is no longer live",
       "live": "Live",
       "offline": "Offline",
-      "preview": "Preview"
+      "preview": "Preview",
+      "unmute": "Unmute",
```

Comment:
don't we have these texts in playkit-ui already ?
if the texts don't exist there - do we have a ticket for adding additional translations ?

Reply:
addressed